### ISSUE TITLE
Radios/Checkboxes with labels are allowed to have aria-labelledby point elsewhere

### ISSUE THREAD

**Issue by colinrotherham** [dwp/govuk-casa#4]
Unfortunately this makes the input lose its original "Example answer 1/2/3" label, they're instead all labelled by the heading above.

![input labels](https://user-images.githubusercontent.com/415517/51914067-df1e0600-23cf-11e9-9ec4-9eb1b9427e78.png)

**Comment by colinrotherham** [comment]
Is it a backwards compatibility feature to allow radio/checkbox inputs to have labels elsewhere?

Perhaps this should be `aria-describedby` instead?

**Comment by colinrotherham** [comment]
Consider removing the `aria-labelledby` test examples, prefer the usual label + input relationship instead https://github.com/dwp/govuk-casa/search?q=%22aria-labelledby%22&unscoped_q=%22aria-labelledby%22

**Comment by lhokktyn** [comment]
Inadvertently closed due to misalignment with internal issue list!

**Comment by colinrotherham** [comment]
This can be closed now, as the `aria-labelledby` examples have been removed. It's up to developers to ensure their HTML markup is accessible.
